sonaranalyzer-dotnet/src/Tests/SonarAnalyzer.UnitTest/TestCases/MethodsShouldNotHaveIdenticalImplementations.vb

```diff
@@ -0,0 +1,93 @@
+Imports System
+
+Namespace Tests.Diagnostics
```

Comment:
Same test cases as for C# (excluding cases that don't apply to VB such as _get_ properties with expressions).